File: js/1866.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
const quantidadeTestes = Number(lines.shift());
for (let i = 0; i < quantidadeTestes; i++) {
    let numeroRepeticoes = Number(lines.shift());
    let soma = 0;
    for (let j = 0; j < numeroRepeticoes; j++) {
        if (j % 2 === 0) {
            soma += 1;
        } else {
            soma -= 1;
        }
    }
    console.log(soma);
}

File: cs/1866.cs
int quantidadeTestes = int.Parse(Console.ReadLine());
for (int i = 0; i < quantidadeTestes; i++)
{
    int numeroRepeticoes = int.Parse(Console.ReadLine());
    int soma = 0;
    for (int j = 0; j < numeroRepeticoes; j++)
    {
        if (j % 2 == 0)
        {
            soma++;
        }
        else
        {
            soma--;
        }
    }
    Console.WriteLine(soma);
}